homeassistant/components/light/__init__.py

```diff
@@ -165,9 +178,15 @@ def handle_light_service(service):
         if transition is not None:
             params[ATTR_TRANSITION] = transition
 
+        service_fun = None
         if service.service == SERVICE_TURN_OFF:
```

Comment:
Yes and no. This would break if we ever changed the value of SERVICE_TOGGLE or SERVICE_TURN_OFF. This probably won't happen soon, but it could happen if we ever wanted to consider internationalization. I have no problem either way as it would be an easy fix when and if that ever happened. It's up to you, really.